src/api-docs.js

```diff
@@ -1625,6 +1678,95 @@ const apiDoc = {
         },
       },
     },
+    '/wallet/atomic-swap/tx-proposal/fetch': {
+      post: {
```

Comment:
Isn't it a get?

Reply:
Yes, it is! This apiDoc wasn't updated after the last review. Fixed it on [595c9a5](https://github.com/HathorNetwork/hathor-wallet-headless/pull/277/commits/595c9a5ffe0cc04e502d7ff1777cd47ba51a4140).